pkg/reconciler/pipelinerun/resources/pipelineref.go

```diff
@@ -176,8 +177,20 @@ func resolvePipeline(ctx context.Context, resolver remote.Resolver, name string)
 // PipelineObject or if there is an error validating or upgrading an
 // older PipelineObject into its v1beta1 equivalent.
```

Comment:
Do we want to update the comment as we are converting v1 to v1beta1 now (and same for taskref)?

Reply:
+1

Reply:
Thanks!